CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Remove `ObjectSerializer#serialized_json` (#91)
 
+### Fixed
+- Ensure caching correctly incorporates fieldset information into the cache key to prevent incorrect cache collisions (#90)
```

Comment:
Let's update the part with `collisions` here as well. See below...

Reply:
Request change has been made.